### PR TITLE
The Indigo Disk Moves

### LINKED ISSUE #3804: Metronome potentially rolls invalid moves + causes freeze
### Description

Metronome appears to be able to roll a invalid move which causes some issues and a game crash.

### Version

1.7.0 (Default)

### Upcoming Version

3a5ca6f8f0b4fd345cf134a04b37361f8f2f776f

### Discord contact info

kakarikiki

### PR DESCRIPTION
**Pull request by Bassoonian**
Adds the Indigo Disk Move data (without all the functionality). Also updates flags of older moves where appropriate (added a wind flag and a Heal Block flag, etc) as well as boosting BP where appropriate. Introduces a config for Sketch bans too.

Fixes #3804 

## Issue(s) that this PR introduces
- Electro Shot
- Tera Starstorm
- Fickle Beam
- Dragon Cheer
- Alluring Voice
- Psychic Noise
- Upper Hand

## **Discord contact info**
bassoonian

### CODE REVIEW

File: src/data/battle_moves.h [hunk 1]
```diff
@@ -14461,6 +14461,77 @@ const struct BattleMove gBattleMoves[MOVES_COUNT_DYNAMAX] =
         .metronomeBanned = TRUE,
     },
 
+    [MOVE_ELECTRO_SHOT] =
+    {
+        .effect = EFFECT_PLACEHOLDER, //EFFECT_ELECTRO_SHOT
```
Comment (AlexOn1ine) [inline src/data/battle_moves.h:14466]: Meteor Beam effect

File: src/data/battle_moves.h [hunk 2]
```diff
@@ -14461,6 +14461,77 @@ const struct BattleMove gBattleMoves[MOVES_COUNT_DYNAMAX] =
...
+        //.secondaryEffectChance = 0,
+        //.target = ,
+        //.priority = 0,
+        .split = SPLIT_PHYSICAL,
+        .zMoveEffect = Z_EFFECT_NONE,
+    },
+
+    [MOVE_THUNDERCLAP] =
+    {
+        .effect = EFFECT_PLACEHOLDER, //EFFECT_ELECTRO_SHOT
```
Comment (AlexOn1ine) [inline src/data/battle_moves.h:14508]: `EFFECT_SUCKER_PUNCH`

File: src/data/battle_moves.h [hunk 3]
```diff
@@ -14461,6 +14461,77 @@ const struct BattleMove gBattleMoves[MOVES_COUNT_DYNAMAX] =
...
+        //.target = ,
+        //.priority = 0,
+        .split = SPLIT_SPECIAL,
+        .zMoveEffect = Z_EFFECT_NONE,
+    },
+
+    [MOVE_TACHYON_CUTTER] =
+    {
+        .effect = EFFECT_HIT,
+        //.power = ,
```
Comment (AlexOn1ine) [inline src/data/battle_moves.h:14523]: 50 BP +  `strikeCount = 2`

File: src/data/battle_moves.h (hunk 1 above)
Comment (AlexOn1ine) [inline src/data/battle_moves.h:14466]: `EFFECT_METEOR_BEAM`

**Review by AlexOn1ine — COMMENTED** [review]
Can we add a spoiler tag and add the rest of the moves?

**Comment by Bassoonian** [comment]
It's the 28th, folks! Ready for review (and to make issues for all missing move effects when we merge this in)

**Review by AlexOn1ine — APPROVED** [review]

**Pull request merged**